### ISSUE TITLE
Enhancing URDF Model

### ISSUE THREAD

**Issue by exo-core** [microsoft/Azure_Kinect_ROS_Driver#237]
I spent some time the last days creating an enhanced URDF description of the Azure based on some actual meshes, as well as adding plugin settings for Gazebo simulation.

![Screenshot from 2022-02-14 17-01-49](https://user-images.githubusercontent.com/35925786/154245340-81b766ee-83ba-419f-8332-ffea123f854e.png)

![Screenshot from 2022-02-16 10-36-28](https://user-images.githubusercontent.com/35925786/154245351-ac11e4a7-a43d-4473-b3ec-421bce0ec198.png)

### Details:
- Created a new URDF descriptions and meshes derived from the CAD model available at [Azure-Kinect-Sensor-SDK](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/tree/develop/assets) (MIT license)
- The meshes were manually optimized in Blender. I removed the hole pattern in the sensor housing to reduce the number of vertices, but kept a few ones to mark the positions of the microphones.
- Included a simplified mesh version for collision model
- The model base is centered on the backmost tripod screw and includes additional coordinate frames for the RGB camera, IR camera and gyro according to the [hardware specifications](https://docs.microsoft.com/en-us/azure/kinect-dk/hardware-specification).
- The new URDF model located in a new file `urdf/azure_kinect_new.urdf.xacro`, while leaving the old one untouched for compatibility
- Gazebo sim settings are made optional
- Usage example:
  ```
  <xacro:include filename="$(find azure_kinect_ros_driver)/urdf/azure_kinect_new.urdf.xacro" />

  <xacro:azure_kinect prefix="" parent="tripod_mount" gazebo="true">
    <origin xyz="0 0 0" rpy="0 0 0" />
  </xacro:azure_kinect>
  ```

The changes can be found in [my fork](https://github.com/tum-i6/Azure_Kinect_ROS_Driver/commits/melodic) of the Azure ROS driver. So far I tested the changes only on Ubuntu and ROS1, but I guess they should also work for ROS2, since far as I know there were no changes to the URDF format.

So long story short: Please let me know whether there is a general interest in creating a PR.

**Comment by ooeygui** [comment]
This looks beautiful! Thank you for sharing. I would definitely have an interest in reviewing a PR.
